README.md

```diff
@@ -44,13 +44,15 @@ Plus, it provides common wrappers around oft-used types like `Function`, `Global
 ## Installation
 
 There are a couple, annoying steps you need to get it working before it'll
-build. Number one, you'll need a custom `cllvm` pkg-config file, which is
-included in the repo. Drop that in `/usr/local/lib/pkgconfig` and make sure
-you have LLVM installed through `homebrew`:
-
-```
-brew install llvm
-```
+build.
+
+- Install LLVM 3.9 using your favorite package manager. For example:
+  - `brew install llvm`
+- Ensure `llvm-config` is in your `PATH`
+  - That will reside in the `/bin` folder wherever your package manager
```

Comment:
It helped me because it didn't end up in my path after installing from homebrew :sad: